rails/bad-paths-can-trigger-invalid_cross_origin_request.md

```diff
@@ -0,0 +1,33 @@
+# Bad JS paths can trigger `InvalidCrossOriginRequest`
+
+In a controller using [`protect_from_forgery`](http://guides.rubyonrails.org/security.html#csrf-countermeasures),
+moving (but not otherwise renaming) a JS file referenced by that
+controller's view(s) without updating the view(s) can trigger an
+`ActionController::InvalidCrossOriginRequest` -- if not in `development`
+or `production`, then perhaps under
+`test` -- though not for the reason shown [here]
+(http://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#csrf-protection-from-remote-script-tags).
+
+In my case, I tracked down that a commit (by someone else, weeks ago...)
+had moved/renamed
+
+    app/assets/javascripts/create_a_class.js
+
+to
+
+    app/assets/javascripts/scorebook/create_a_class.js
+
+It seemed to work fine in `development`, but my new Capybara `feature`
+spec triggered
+
+    Failure/Error: Unable to find matching line from backtrace
+    ActionController::InvalidCrossOriginRequest:
+      Security warning: an embedded <script> tag on another site requested protected JavaScript. If you know what you're doing, go ahead and disable forgery protection on this action to permit cross-origin JavaScript embedding.
```

Comment:
Can we break this up into multiple lines to there is no horizontal scrolling to read the error message?